packages/web/test/src/lib/walletconnect-mode.ts

```diff
@@ -0,0 +1,111 @@
+import WalletConnect from "@walletconnect/client";
+import algosdk, { Account, Transaction } from "algosdk";
+import assert from "assert";
+import QRCodeModal from "algorand-walletconnect-qrcode-modal";
+import { testnetURL, types, WallectConnectSession } from "../../../src";
+import { algoexplorerAlgod, getSuggestedParams } from "../../../src/lib/api";
+import { HttpNetworkConfig } from "../../../src/types";
+
+describe("Webmode - Wallet Connect test cases ", function () {
+    let connector: WallectConnectSession;
+    let sender: Account;
+    let receiver: Account;
+
+    let algodClient: algosdk.Algodv2;
+    const walletURL: HttpNetworkConfig = {
+        token: "",
+        server: testnetURL,
+        port: "",
+    }
+    algodClient = algoexplorerAlgod(walletURL);
+
+    this.beforeEach(function () {
+        sender = algosdk.generateAccount();
+        receiver = algosdk.generateAccount();
+        connector = new WallectConnectSession(walletURL, new WalletConnect({
+            bridge: "https://bridge.walletconnect.org",
```

Comment:
If it's already declared as constant then please use it else create one.

Reply:
it is not declared as constant, and it shouldn't be declared one as we create the instance in `beforeEach`